deltachat-ios/Controller/ChatListViewController.swift

```diff
@@ -737,6 +765,7 @@ class ChatListViewController: UITableViewController {
             if !isArchive {
                 navigationItem.setLeftBarButton(cancelButton, animated: true)
             }
+            navigationItem.setRightBarButtonItems([newButton], animated: true)
```

Comment:
there is no "new chat" option in the archive when _not_ forwarding, so we should not have one in special forwarding-mode as well:

```suggestion
            navigationItem.setRightBarButtonItems(isArchive ? nil : [newButton], animated: true)
```